WikiDrips/WikiTableViewController.swift

```diff
@@ -179,16 +196,16 @@ extension WikiTableViewController: UITableViewDataSourcePrefetching {
         }
     }
     
-// No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
+    // No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
```

Comment:
So sounds to me like something I should handle in a pr that adds the ability to retrieve more than 10 records.  A prefetch for the last record could call the Wikipedia api for the next set of 10 records.  Sound right?